README.md

```diff
@@ -1,2 +1,224 @@
+
+**Setting desired capabilities**
+
+Add there environment variables to CircleCI Environment Variables configuration window:
+- `KOBITON_DEVICE_NAME`: Kobiton device name (`deviceName` in the generated desired capabilities).
+- `KOBITON_DEVICE_PLATFORM_NAME`: Kobiton device paltform name (`platformName` in the generated desired capabilities).
+- `KOBITON_DEVICE_PLATFORM_VERSION`: Kobiton device platform version (`platformVersion` in the generated desired capabilities).
+
+For example, if desired capabilities for executing automation test on `Pixel 2 XL` running `Android 8.1.0`:
+
+```javascript
+const desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: 'Demo Automation Test on Android', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<APP_URL>',
+  deviceGroup:        'KOBITON', 
+  deviceName:         'Pixel 2 XL',
+  platformVersion:    '8.1.0',
+  platformName:       'Android' 
+}
+```
+
+Then the corresponding environment variables set in CircleCI should be
+
+![Example Android DesiredCaps](./docs/assets/desired-caps-circleci-env.png)
+
+> More information about Desired Capabilities and its parameters can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+#### 4. CircleCI configuration file
+**Execution environment**
+
+As the automation testing script is written in NodeJS, which requires the execution environment to have NodeJS installed, therefore, we will be using CircleCI official latest NodeJS container `circleci/node:latest` as the execution environment.
+
+In the provided CircleCI configuration file `/.circleci/config.yml`, the execution environment has already been set to using the mentioned container.
+
+```yaml
+docker:
+    - image: circleci/node:latest
+```
+
+> For more information about how to execute automation test(s) on Kobiton, you can visit:
+> - Kobiton documentation: https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/
+> - Kobiton's samples for automation test scripts in other languages: https://github.com/kobiton/samples
+
+**Automation test script execution**
+
+To execute the automation test script, just add a `run` section with command `npm run <SCRIPT_FILENAME>`. By default, it has already been added to the provided CircleCI configuration file.
+
+Example of executing the provided automation script:
+
+```yaml
+- run:
+    name: app-auto-testing
+    command: npm run automation-test-script
+    working_directory: samples/automation-test
+```
+
+### 5. Configuring HockeyApp WebHook listener
+Kobiton has provided script for setting up a small, basic server for receiving WebHook notifications from HockeyApp and triggering CircleCI to execute automation test of that application on Kobiton. The script will load settings from a configuration file and start a WebHook listener. This script must be executed on the server mentioned in **Prerequisites**.
```

Comment:
Please highlight this (bolding, write in a new line,...). This line is essential yet is too easy to miss.
```
This script must be executed on the server mentioned in **Prerequisites**.
```